src/components/calcite-label/calcite-label.scss

```diff
@@ -13,71 +13,69 @@
 }
 
 :host([scale="s"]) {
-  --calcite-label-margin-bottom: 12px;
+  --calcite-label-margin-bottom: theme("spacing.3");
   label {
     @apply text--2;
   }
 }
 :host([scale="m"]) {
-  --calcite-label-margin-bottom: 14px;
+  --calcite-label-margin-bottom: theme("spacing[3.5]");
```

Comment:
@jrowlingson so are we using the 1.x or the 2.x spacing scale?

Reply:
I used these values to match what was there, there is no label component in Figma. 

@bstifle, maybe you can advise on spacing here.

Reply:
I don't think in our current spacing scale 3.5 is available is why I asked. This may need a new value.

Reply:
We'll want to use the 1.x scale and continue to do so once we've upgraded to 2.x. Preference is for the more constrained scale.

Reply:
GONE! using 2, 3, 4